lms/static/lms/js/xblock/lms.runtime.v1.js

```diff
@@ -0,0 +1,55 @@
+(function(URI) {
+    'use strict';
+
+    var __hasProp = {}.hasOwnProperty,
+        __extends = function(child, parent) {
+            var key;
+            for (key in parent) {
+                if (__hasProp.call(parent, key)) {
+                    child[key] = parent[key];
+                }
+            }
+            function Ctor() {
+                this.constructor = child;
+            }
+            Ctor.prototype = parent.prototype;
+            child.prototype = new Ctor();
+            child.__super__ = parent.prototype;
+            return child;
+        };
```

Comment:
Coffeescript _loves_ putting this block into every file it generates, but I can't quite figure out what exactly it does. Any idea? Do we really need it in every case?